packages/grid/data-grid/src/DataGrid.tsx

```diff
@@ -36,6 +37,7 @@ export type DataGridProps = Omit<
   pagination?: true;
   apiRef?: undefined;
   onRowsScrollEnd?: undefined;
+  selectionModel?: GridRowId | GridRowId[];
```

Comment:
Should we update https://deploy-preview-1823--material-ui-x.netlify.app/api/data-grid/data-grid/?

Reply:
yes done